addons/account/models/account_journal.py

```diff
@@ -807,5 +807,5 @@ def _get_last_bank_statement(self, domain=None):
         '''
         self.ensure_one()
         last_statement_domain = (domain or []) + [('journal_id', '=', self.id)]
-        last_st_line = self.env['account.bank.statement.line'].search(last_statement_domain, order='date desc, id desc', limit=1)
-        return last_st_line.statement_id
+        last_statement = self.env['account.bank.statement'].search(last_statement_domain, order='date desc, id desc', limit=1)
```

Comment:
```suggestion
        last_statement = self.env['account.bank.statement'].search(last_statement_domain, limit=1)
```
The order is, I think, no longer needed here as it would use the one defined on the model. 
Date > name > id. Unless we don't want to care about the name I guess.

Reply:
I think the name does not matter, and even make the order inefficient in this case. As we want the last statement created, if it's the same date, id is the second parameter to take into account

Reply:
Taking the name makes this fix inefficient, I already tried.
If the statement has the same date, the order would be completely dependant of the name, or we want to have the last statement created by user.

Reply:
Not too sure about that. Nothing guarantees you the user creates the statements in a chronologic order.

Reply:
About the fix? Or we should not fix the current behaviour?